### PR TITLE
feat(project): Adding examples - OEL-1048

### PR DESCRIPTION
**Pull request by tibi2303**

### CODE REVIEW

File: src/compositions/bcl-project/details.html.twig [hunk 1]
```diff
@@ -0,0 +1,69 @@
+{% extends '@oe-bcl/bcl-base-templates/content-type.html.twig' %}
+
+{% block sidebar %}
+  {% include '@oe-bcl/bcl-inpage-navigation/inpage-navigation.html.twig' with sidebar only %}
+{% endblock %}
+
+{% block content %}
+  <h3 class="fw-bold mb-4">Project details</h3>
```
Comment (planctus) [inline src/compositions/bcl-project/details.html.twig:8]: Oh gosh, you did it!! you made the widget with a  "composition", really cool, but then:
why don't you really create a composition, a template with params and then you can really use it in the real world to render that widget..?

File: src/compositions/bcl-project/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@openeuropa/bcl-project",
+  "author": "European Commission",
+  "license": "EUPL-1.2",
+  "version": "0.18.0",
+  "description": "OE - BCL Project",
+  "publishConfig": {
+    "access": "public"
+  },
```
Comment (planctus) [inline src/compositions/bcl-project/package.json:9]: it's not really relevant, but we should add dependencies for all the templates we include in the package templates

File: src/components/bcl-progress/progress.html.twig [hunk 3]
```diff
@@ -6,6 +6,7 @@
   - max (int) (default: 100)
```
Comment (planctus) [inline src/components/bcl-progress/progress.html.twig:6]: since you did everything in this pr, could you rename it..? :)

Comment (tibi2303) [reply src/components/bcl-progress/progress.html.twig:6]: done.

File: src/compositions/bcl-project/package.json (hunk 2 above)
Comment (tibi2303) [reply src/compositions/bcl-project/package.json:9]: done, also updated the twig templates package.

File: src/compositions/bcl-project/details.html.twig (hunk 1 above)
Comment (tibi2303) [reply src/compositions/bcl-project/details.html.twig]: done, hope it's ok now.

File: src/compositions/bcl-project/project.story.js [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+  },
+};
+
+export const Listing = () => listingPage(correctPaths(demoListing));
+Listing.storyName = "Listing";
+Listing.decorators = [initBadges];
+
+export const DetailsOngoing = () =>
+  detailsPage(correctPaths(demoDetails, "default"));
+DetailsOngoing.storyName = "Details Ongoing";
```
Comment (planctus) [inline src/compositions/bcl-project/project.story.js:132]: We are not very consistent with this naming, in general we name these "name of the feature" page, here i would set Project page (ongoing)
Project page (planned)
Project page (closed)

File: src/compositions/bcl-event/data/event-sidebar.js [hunk 5]
```diff
@@ -25,7 +25,14 @@ module.exports = {
           path: "/icons.svg",
         },
       },
-      definition: ["From: 19/122/2021", "To: 21/12/2021"],
+      definition: [
```
[the remark this answers is not in the document]
Comment (tibi2303) [reply src/compositions/bcl-event/data/event-sidebar.js]: I removed these changes

**Pull request merged**